### PR TITLE
Document verified_at Attribute

### PR DESCRIPTION
**Pull request by zachmargolis**
Document changes added by

https://github.com/18F/identity-idp/pull/3602

### CODE REVIEW

File: _pages/attributes.md [hunk 1]
```diff
@@ -6,13 +6,158 @@ lead: >
 
 Here are the possible attributes that can be requested at a given IAL. This table contains the available user attributes, the IAL they are associated with, and how they can be accessed in OpenID Connect and SAML.
 
-| Attribute | IAL1 | IAL2 | OpenID Connect | SAML |
```
Comment (jmhooper) [inline _pages/attributes.md]: No love for markdown tables here

**Comment by jmhooper** [comment]
Hmm, it looks like Federalist blew up on the build. I logged into federalist and it looks like the build doesn't have any logs associated with it 🤔

File: _pages/attributes.md (hunk 1 above)
Comment (jmhooper) [reply _pages/attributes.md]: It looks like the `verified_at` attribute did not actually get added to this table. Was that intentional?

**Comment by jmhooper** [comment]
I queued up another build. Looks like that one was successful.

File: _pages/attributes.md (hunk 1 above)
Comment (zachmargolis) [reply _pages/attributes.md]: Well I do like Markdown tables a lot but this was getting tough to read and edit.

And yes I forgot to add the attribute after I converted it :see_no_evil:

Comment (zachmargolis) [reply _pages/attributes.md]: Fixed in 491c75a50f869fea1ce00d6bfefc25639d837dd6

**Pull request merged**